packages/web-components/src/theme/theme.stories.ts

```diff
@@ -0,0 +1,24 @@
+import { DesignToken } from '@microsoft/fast-foundation';
+import * as tokens from '../theme/design-tokens';
+
+DesignToken.registerDefaultStyleTarget();
+
+export default {
+  title: 'Theme/Tokens',
+};
+
+export const Tokens = () => `
+<div>
+  <h3>Theme Tokens</h3>
+  <p>Debug story which uses theme tokens to style the element below.</p>
+  <div style="
+    font-family: ${tokens.fontFamilyBase.createCSS()};
```

Comment:
@nicholasrice What do you think about adding a `toString()` and `valueOf()` implementation to the design to `CSSDesignToken`? That would make this syntax nicer if used outside of a FAST `html` or `css` template.

Reply:
`toString()` seems fine to me. Can you elaborate more on `valueOf()`? It's not clear to me what that would return.

Reply:
Actually, I don't think we need `valueOf()`. I just checked and the JavaScript runtime prefers `toString()` for string conversions over `valueOf()`. Since we don't need to convert to a numerical type of some non-string type we should be fine with `toString()`.

Reply:
Oh I see, I didn't realize that was an object prototype method. I think adding `toString()` to `CSSDesignToken` makes sense.